LPS-30949 Regenerate

brandizzi opened the pull request.

brandizzi committed:
LPS-30949 Asking user about updating either only the current booking or all children bookings

Some changes include
- Untangling persistence service calls, remote consultation invokations and dialog creation
- Extracting confirmation dialog to function and using it to get user choices in edit window
- Changing messages on deletion (because we do not delete events, just move to trash)

brandizzi committed:
LPS-30949 Avoiding duplication of children events

brandizzi committed:
LPS-30949 Updating services for allowing propagation of edits to child events

- Adding boolean parameter to service methods about whether should update
  children events
- Exposing service method for counting children. Using it instead of imprecise
  search

brandizzi committed:
LPS-30949 "Cleaning up" small issues on frontend before proceeding

- Using more reasonable dimensions to confirm dialog

- Partial revert of 79fd9c85: enabling invited users to edit time

- Revert "LPS-30949 Preventing invited users from changing time from children events"
  This reverts commit 8d5dff686665e127ba79e7ea3f844ef879d9d9b3.

- Removing broken memoization from confirm dialogs

eduardolundgren commented:
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

brandizzi committed:
LPS-30949 Liferay.CalendarList accepts only one instance of each calendar

This avoids duplication of calendars on the list of invited resources of the booking edit window
when there is more than one child calendar booking

brandizzi committed:
LPS-30949 Regenerate

eduardolundgren commented:
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

eduardolundgren commented:
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._
